pyproject.toml

```diff
@@ -1,13 +1,14 @@
-[tool.poetry]
++[tool.poetry]
```

Comment:
⚠️ 
```suggestion
[tool.poetry]
```

Reply:
I also noticed it. It will be changed as the project no longer runs with this typo.